docs/manual/platforms/xbox-one.md

```diff
@@ -1,40 +1,41 @@
-4. In the second popup - called `Choose any necessary dependencies` choose `Microsoft.VCLibs..appx` and then click `Start`
-5. In your PC go to the shared folder `WindowsApps` and find the devilutionX folder which should be something like `<random string>\_<version>\_random string<>`
-6. Copy `diabdat.mpq` from your CD (or GoG install folder) to it.
-7. In the Xbox press `View` on DevilutionX and select `Details` and there change the type from `App` to `Game`.
-
-## Usage
-
-Launch `DevilutionX` from the Dev Mode home.
-
-## Controls
-
-- Left analog or D-Pad: move hero
-- A: attack nearby enemies, talk to townspeople and merchants, pickup/place items in the inventory, OK while in main menu
-- B: select spell, back while in menus
-- X: pickup items, open nearby chests and doors, use item in the inventory
-- Y: cast spell, delete character while in main menu
-- LB: use health item from belt
-- RB: use mana potion from belt
-- LT: character sheet (alt: Menu + LB or ←)
-- RT: inventory (alt: Menu + RB or →)
-- Left analog click: toggle automap (alt: Menu + ↓)
-- Menu + View: game menu (alt: Menu + ↑)
-- View + A/B/X/Y: Spell hotkeys
-- Right analog: move automap or simulate mouse
-- Right analog click: left mouse click (alt: View + LB)
-- View + Right analog click: right mouse click (alt: View + RB)
-- View + LT: quest log (alt: Menu + Y)
-- View + RT: spell book (alt: Menu + B)
-
-## Resources
-
-* Discord: https://discord.gg/YQKCAYQ
-* GitHub: https://github.com/diasurgical/devilutionX
+# DevilutionX (Diablo 1) for Microsoft Xbox One/Series
+
+## Installation
+
+1. Download and unzip [devilutionx-xbox-one.zip](https://github.com/diasurgical/devilutionX/releases/latest/download/devilutionx-xbox-one.zip).
+2. In Xbox Device Portal click on `Add` button located at `Home/My games & apps`
+3. In the first popup - called `Deploy or Install Application` choose `devilutionX.appx` and then click `Next`
+4. In the second popup - called `Choose any necessary dependencies` choose `Microsoft.VCLibs.appx` and then click `Start`
+5. In your PC go to the shared folder `WindowsApps` and find the devilutionX folder which should be something like `<random string>\_<version>\_random string<>`
+6. Copy `diabdat.mpq` from your CD (or GoG install folder) to it.
+7. In the Xbox press `View` on DevilutionX and select `Details` and there change the type from `App` to `Game`.
+
+## Usage
+
+Launch `DevilutionX` from the Dev Mode home.
+
+## Controls
+
+- Left analog or D-Pad: move hero
+- A: attack nearby enemies, talk to townspeople and merchants, pickup/place items in the inventory, OK while in main menu
+- B: select spell, back while in menus
+- X: pickup items, open nearby chests and doors, use item in the inventory
+- Y: cast spell, delete character while in main menu
+- LB: use health item from belt
+- RB: use mana potion from belt
+- LT + ←: character sheet

```

Comment:
A completely different control scheme and implementation only for UWP seems rather hard to maintain.

Reply:
Another solution would keeping the current implementation for limited systems like vita and 3ds and call the one I'm adding "full range" and use it for everything else. So PC, Mac, Xbox and switch would use this configuration as those have access to all needed buttons

Reply:
https://github.com/diasurgical/devilutionX/pull/3975/commits/a0a5f39cb86d8f5467a0d6e100b2a7d0ce9df7df
:point_up: this does the trick